package.json

```diff
@@ -8,10 +8,11 @@
   "source": "./src/index.ts",
   "types": "./dist/index.d.ts",
   "exports": {
-    "types": "./dist/index.d.ts",
-    "import": "./dist/index.js",
-    "require": null,
-    "default": "./dist/index.js"
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "default": "./dist/index.js"
+    }
```

Comment:
Why the change?

Reply:
I am not sure, but in this version it is using subpath exports: https://nodejs.org/api/packages.html#subpath-exports in case we want to export other entry points later ?